repos/spack_repo/builtin/packages/rocm_tensile/package.py

```diff
@@ -97,7 +97,7 @@ class RocmTensile(CMakePackage):
 
     root_cmakelists_dir = "Tensile/Source"
 
-    patch("0003-require-openmp-extras-when-tensile-use-openmp.patch", when="@5.6.0:")
+    patch("0003-require-openmp-extras-when-tensile-use-openmp.patch", when="@5.7.0:")
```

Comment:
It seems this applies to all versions:
```suggestion
    patch("0003-require-openmp-extras-when-tensile-use-openmp.patch")
```